runtime/src/accounts_db.rs

```diff
@@ -2228,6 +2396,23 @@ pub mod tests {
             }
         }
 
+        fn append_vec_count_for_slot(&self, slot: Slot) -> usize {
```

Comment:
Isn't this the accounts count for the slot?

Reply:
Err, the code is correct and the fn name is confusing... I've fixed that!